### ISSUE TITLE
Kobo Nia original SIM card

### ISSUE THREAD

**Issue by S-Pasta** [pgaskin/KoboStuff#41]
After connecting the charger from the laptop and switching to reading files, Nia stopped displaying anything. Is it possible to download the original system and replace it on the SIM card?

**Comment by pgaskin** [comment]
You mean the SD card, and ask on the MobileRead Forums.

First, though, have you tried turning it off? Hold the power button for 30 seconds with the Nia unplugged, then let go, then do it again.

**Comment by S-Pasta** [comment]
Yes, I did these steps, but Kobo does not respond to restarting and still does not display anything.
I believe that only uploading the digital image of the SD card again will fix the problem. 
Hence my search for the original Kobo Nia SD card image

**Comment by pgaskin** [comment]
Post a message [here](https://www.mobileread.com/forums/showthread.php?t=355778).

**Comment by S-Pasta** [comment]
Tak, wykonałem te kroki, ale Kobo nie reaguje na ponowne uruchomienie i nadal nic nie wyświetla.
Wierzę, że dopiero ponowne przesłanie cyfrowego obrazu karty SD rozwiąże problem. Stąd moje poszukiwania oryginalnego obrazu karty SD Kobo Nia
